spec/controllers/prior_authority/reason_why_controller_spec.rb

```diff
@@ -1,90 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe PriorAuthority::Steps::ReasonWhyController, type: :controller do
-  before do
-    allow(controller).to receive(:current_application).and_return(current_application)
-  end
-
-  describe '#create' do
```

Comment:
How come these specs have all been removed? Did they break? I wouldn't have expected reason why controller's behaviour to have changed.

Reply:
sorry I was under the impression the code I was bringing in from Joel's branch was an update to this, will look into this now

Reply:
sorry this comment was relative to the pull_updates_spec tests. These tests haven't been removed, they are run in the shared example